run-accessibility-check.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+# Script to run the Accessibility Check locally
+# Constants to add colour to text
+BLUE='\033[0;34m'
+NC='\033[0m' # No Color
+# Build docker (can comment out if already built)
+docker-compose build

```

Comment:
Option 2 (Part 1 of 2): replace with 'docker-compose up --d --build' to bring it up in the background and force a build